FBSnapshotsViewerTests/ApplicationLogReaderSpec.swift

```diff
@@ -15,7 +15,7 @@ import Foundation
 class ApplicationLogReaderrSpec: QuickSpec {
     override func spec() {
         var reader: ApplicationLogReader!
-        describe(".readline(of:startingFrom:)") {
+        fdescribe(".readline(of:startingFrom:)") {
```

Comment:
nope. this won't go :)

Reply:
This is definitely a good catch, I think I need to automate it with Danger.

Reply:
@PaulTaykalo would be good if you can approve the PR, so I will deliver the new version.